File: c & c++/1020.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
	
		int a = 0;
		int m = 0;
		int d = 0;

		int x = 0;

		int z = 0;

		scanf("%d" , &x);

		z = x / 365;

		if(z > 0){
			a = z;
			z = 0;
			x %= 365;
		}

		z = x / 30;

		if (z > 0)
		{
			m = z;
			z = 0;
			x %= 30;	
		}

		d = x;

		printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", a ,m , d );


	return 0;
}

File: go/1020.go
package '1020'

import (
	"fmt"
)

func main() {
	var days int

	fmt.Scanf("%d", &days)

	years := days / 365
	years_rest := days % 365
	months := years_rest / 30
	montts_rest := years_rest % 30

	fmt.Printf("%v ano(s)\n", years)
	fmt.Printf("%v mes(es)\n", months)
	fmt.Printf("%v dia(s)\n", montts_rest)
}
